### DISCUSSION TITLE
Adjust color on homepage to match zen theme

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#826] [Ideas]
### What feature would you like to see?

The landing/homepage/newtab page has the custom zen logo, but the colorscheme isnt matched.
In firefox its matched to the rest of the UI, when using zed, its still firefox colors and not matched to the theme
<img width="593" alt="{922AC6FF-B8BA-4B89-862F-477B2038161C}" src="https://github.com/user-attachments/assets/841c2f7f-2594-4125-9cc3-ff353949731c">
<img width="1131" alt="{A5191651-5C64-4A64-BF86-BB0446EDD002}" src="https://github.com/user-attachments/assets/c112f2a0-70b5-4658-b1b1-d6cad4594187">

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/480_

**Comment by aheartonaheart** [comment]
girls in stem would really appreciate you doing this <3 (im girls in stem)
